.travis.yml

```
@@ -132,9 +132,9 @@ jobs:
       name: Subcommands on Kubernetes
       script: make test-subcommand
 
-    # Build and test go
+    # Build and test go for current project layouts
```

Nit: `current` will have new meaning post `v0.18.0`. So maybe use `legacy` to refer to the current layout like we're doing elsewhere in our comments.
```suggestion
    # Build and test go for legacy project layouts
```